api/src/main/java/io/minio/CopySource.java

```diff
@@ -0,0 +1,136 @@
+package io.minio;
+
+import java.util.HashMap;
+import java.util.Map;
+import io.minio.errors.InvalidArgumentException;
+
+/** SourceInfo creates the SourceObject
+ * used for Compose Object.
```

Comment:
Is this comment correct?

Reply:
Modified.

Reply:
This comment is not useful or not clear.